battle/effect_commands.asm

```diff
@@ -730,7 +730,7 @@ BattleCommand_CheckObedience: ; 343db
 
 	; risingbadge
 	bit RISINGBADGE, [hl]
-	ld a, MAX_LEVEL + 1
+	ld a, LOW(MAX_LEVEL + 1)
```

Comment:
This could cause nothing to obey if the max level was 255. `MAX(MAX_LEVEL, $ff)` would at least allow anything below 255 to obey. Not sure if `MAX()` exists.

Reply:
I think in the other case the way the code was written allowed it to work. It's ok to error if it can't easily work.